OKR.md

```diff
@@ -5,22 +5,22 @@ We try to frame our ongoing work using a process based on quarterly Objectives a
 ## 2018 Q4
 
 **Debuggability and Stability**
-- error handling is done correctly and error codes are part of the API - @jacobheun
-- libp2p has a network visualization tool - @jacobheun
-- js-libp2p has a daemon with a RPC API and a client library - @jacobheun
-- interop tests for go-libp2p and js-libp2p exist for core functionality - @vasco-santos
-- a testbed exists and can test 50,000 libp2p nodes (browser and node) - @jacobheun
-- the testbed can run large scale tests for the DHT and Pubsub - @vasco-santos
+- [P2] error handling is done correctly and error codes are part of the API - @jacobheun
+- [P1] libp2p has a network visualization tool - @jacobheun
+- [P1] js-libp2p has a daemon with a RPC API and a client library - @jacobheun
```

Comment:
I would consider this to be a P0, as it has several dependencies. What do you think @jacobheun ?

I agree with the other priorities.

Reply:
Yes, I think you're right, as it's a blocker for some of the other P0s. I'll update.